modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/exportimport/data/handler/VulcanBatchEnginePortletDataHandler.java

```diff
@@ -138,6 +139,18 @@ protected PortletPreferences doImportData(
 			return portletPreferences;
 		}
 
+		String userIdStrategy = StringPool.BLANK;
+
+		if (UserIdStrategy.CURRENT_USER_ID.equals(
+				MapUtil.getString(
+					portletDataContext.getParameterMap(),
```

Comment:
Just a check (might be just fine): is it always sure that the parameter map is not null? MapUtil doesn't do null check for the map itself.